src/components/mdxComponents/cloud-link.tsx

```diff
@@ -35,12 +36,44 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-interface Props {
+interface Props { }
+
+export const CliDeployInstructions: React.FC<Props> = () => {
```

Comment:
Maybe you can make this a generic `TemplatedCodeBlock` component which takes the desired commands a prop:
```
<TemplatedCodeBlock 
    code={`
       # List Pixie deployment options.
        px deploy --help
        # Deploy the Pixie Platform in your K8s cluster (No OLM present on cluster).
        export PL_CLOUD_ADDR=@@`
     }
/>
```

Let me know if that makes sense/if you need any help with that.

Reply:
Thanks for this suggestion! I didn't realize that passing as a prop would be different than accepting the string from within a component's tags.

I've updated this to reflect this and also made changes to the Helm and YAML install pages as well. I would recommend reviewing my later changes commit by commit since I made another few adjustments along the way: renaming `@@` to `@PLACEHOLDER` and renaming `PL_` -> `PX_`.